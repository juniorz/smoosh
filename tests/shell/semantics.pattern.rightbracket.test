touch file]
touch filea

echo file[]123]
echo file[[.].]]
echo file[[=]=]]
echo file[!]123]
echo file[[:alpha:]]
echo file[a-z]