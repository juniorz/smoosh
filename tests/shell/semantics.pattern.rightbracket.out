file]
file]
file]
filea
filea
filea
